.circleci/config.yml

```diff
@@ -1,26 +1,39 @@
 version: 2
+
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:10
 
     working_directory: ~/repo
 
     steps:
       - checkout
 
-      # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "package.json" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
+            - npm-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - npm-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - npm- # used if checksum fails
 
       - run: npm install
 
       - save_cache:
+          key: npm-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
           paths:
-            - node_modules
-          key: v1-dependencies-{{ checksum "package.json" }}
+            - node_modules/
 
       - run: npm test
+
+      # Tried using npx, but there are weird issues when uploading.
+      - run:
+          name: codecov
```

Comment:
Why Codecov over Coveralls? Do we have subscriptions with both?

Reply:
Simply because the GitHub App for Codecov was already installed.

Reply:
Ok, looks nicer than Coveralls anyway :grin: